src/fiatconnect/saga.test.ts

```diff
@@ -773,6 +784,14 @@ describe('Fiatconnect saga', () => {
     })
     it('saves an error when providers is null and fetching them fails', async () => {
       jest.mocked(fetchQuotes).mockResolvedValue(mockFiatConnectQuotes)
+      jest.mocked(getDynamicConfigParams).mockImplementation(({ configName }) => {
+        if (configName === StatsigDynamicConfigs.FIAT_CONNECT_CONFIG) {
+          return {
+            fiatConnectCashOutEnabled: true,
+          }
+        }
+        return {} as any
+      })
```

Comment:
perhaps moving this to the top level `describe` removes the need to re-mock this in every test group?